modules/ossm-security-mtls-1x.adoc

```diff
@@ -70,7 +70,7 @@ If your environment has specific requirements for encrypted traffic in your serv
 
 [source,yaml]
 ----
-apiVersion: maistra.io/v2
+apiVersion: maistra.io/v1
```

Comment:
The current YAML example is against the OSSM v2 SMCP schema.  I had to dig around in the archives to find the proper YAML for a v1 SCMP.    Here it is.

[source,yaml]
----
apiVersion: maistra.io/v1
kind: ServiceMeshControlPlane
spec:
  istio:
    global:
      tls:
        minProtocolVersion: TLSv1_2
        maxProtocolVersion: TLSv1_3
----

Reply:
@JStickler thanks for digging into the old v1 SMCP YAML file. I've updated the source and the preview build is available.